LayoutTests/TestExpectations

```diff
@@ -2274,6 +2274,9 @@ fast/images/image-overlay-mutiple-frames.html [ Skip ]
 # These tests don't have to be platform-specific, but they are only implemented on Mac now.
 fast/images/eps-as-image.html [ Skip ]
 
+# Skip this test to avoid flickering due to multiple async image decoding for different sizeForDrawing.
```

Comment:
Is there a bug tracking this skipping? What do we need to do to unskip?

Reply:
No tests will skipped. A new layout test was added.